### ISSUE TITLE
Adjust anchor points for chapters and scenes in annotation windows

### ISSUE THREAD

**Issue by dcheather** [yale-web-technologies/mirador-project#143]
Change the anchor points from chapter and scenes in the annotation window to above the black bars, not below them.

**Comment by gigamorph** [comment]
ETA: 7/7/2017

**Comment by dcheather** [comment]
The chapter select list at the top of the annotations window is not showing in staging. Is that going to be added back in?

**Comment by gigamorph** [comment]
No, we didn't want that anymore, do we?

**Comment by dcheather** [comment]
We do, see ticket #132.

**Comment by gigamorph** [comment]
Yes, but I also think Andy didn't want it once the side panel is available. Can we ask him?

**Comment by gigamorph** [comment]
Well, it's easy to put it back in. Maybe I will for the moment.

**Comment by dcheather** [comment]
If Andy wanted it removed, he didn't tell me. I haven't had any communication with him since I delivered the requirements. I will have Dean follow up with him.

**Comment by gigamorph** [comment]
Done.

**Comment by gigamorph** [comment]
Have we heard from Andy about this issue? Should I just re-enable the selector?

**Comment by schneblehead** [comment]
He did not have a strong feeling either way since that functionality is available on the TOC panel. Let's leave it as-is for now.  I will schedule a meeting with him next week when he's back and we can discuss it then.
